modular_exponential/modular_exponential.py

```diff
@@ -0,0 +1,21 @@
+def mod_exponent(base, power, mod):
+    res = 1  # Initialize result
```

Comment:
Function docstring should be added according to me describing param's and return type